### PR TITLE
Add regex for IBANs

### PR DESCRIPTION
**Pull request by fakenine**
Bonjour!

I added support for IBANs, along with tests.

### CODE REVIEW

File: commonregex_test.go [hunk 1]
```diff
@@ -567,3 +567,30 @@ func TestCommonRegex_MACAddresses(t *testing.T) {
 		}
 	}
 }
+
+func TestCommonRegex_IBAN(t *testing.T) {
```
Comment (mingrammer) [inline commonregex_test.go]: Use `TestCommonRegex_IBANs` instead of `TestCommonRegex_IBAN` please :)

**Comment by fakenine** [comment]
@mingrammer Done :)

**Comment by mingrammer** [comment]
Thanks for contributing :)

**Comment by fakenine** [comment]
You're welcome :) Just noticed I forgot to update the `README.md`. Do you need another PR for it ?

**Comment by mingrammer** [comment]
@fakenine Oh, that's right. Could you do for it? :)

**Pull request merged**